crates/bevy_animation/src/lib.rs

```diff
@@ -171,6 +178,11 @@ impl AnimationPlayer {
         self.elapsed = elapsed;
         self
     }
+
+    /// Handle to the animation clip.
+    pub fn animation_clip(&self) -> &Option<Handle<AnimationClip>> {
+        &self.animation_clip
+    }
```

Comment:
```suggestion
    pub fn animation_clip(&self) -> Option<&Handle<AnimationClip>> {
        self.animation_clip.as_ref()
    }
```
Returning `Option<&T>` is preferable.

Reply:
Ok, I can change that.